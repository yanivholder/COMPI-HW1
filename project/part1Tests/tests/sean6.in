"all the special characters \n\t\0\" without carriage return"