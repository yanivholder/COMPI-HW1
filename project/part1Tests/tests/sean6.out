1 STRING all the special characters 
	
